templates/hazelcast5-existing-cluster/README.md

```diff
@@ -0,0 +1,88 @@
+# Hazelcast Simulator – Existing Cluster Template
```

Comment:
Do we really need yet another template or could we have just added an optıon to existing `hazelcast5-lab` template? Can we avoid duplications? Is it worth really adding a new template?